docs/reference/publication_formats.md

```diff
@@ -1,12 +1,152 @@
 # Publication formats reference
 
+```{admonition} Alpha consultation
+The following issues relate to this page:
+* [#51 Packaging multiple networks](https://github.com/Open-Telecoms-Data/open-fibre-data-standard/issues/51)
+```
+
+OFDS data must be published using at least one of the formats described on this page:
+
+* [JSON](#json)
+* [GeoJSON](#geojson)
+* [CSV](#csv)
+
+Each format provides containers for publishing one or more networks and options to support pagination and streaming.
+
+To support the widest range of use cases, you should publish your data in all three formats. For more information on choosing a publication format and on publishing data in multiple formats, see [how to format data for publication](../guidance/publication.md#how-to-format-data-for-publication).
+
 ## JSON
 
+This section describes how to publish data in JSON format.
+
+If your data is small enough to fit into memory or if you are publishing data via an API, you should use the [small files and API responses option](#small-files-and-api-responses-option). If your data is too large to fit into memory, you should use the [streaming option](#streaming-option).
+
+For guidance on paginating or streaming individual networks, see [how to publish large networks](../guidance/publication.md#how-to-publish-large-networks).
+
+### Small files and API responses option
+
+The network package schema describes the structure of the container for publishing one or more networks in JSON format and for supporting pagination.
+
+For this version of OFDS, the canonical URL of the schema is [https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-package-schema.json](https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-package-schema.json). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
+
+This page presents the schema in an interactive browser, you can also download the canonical version of the schema as [JSON Schema](../../schema/network-package-schema.json).
+
+A network package is a JSON object that must include `.networks`, an array of `Network` objects as described by the [network schema](schema.md). For data published via a paginated API, the optional `.pages` object should be used to provide links to the next and previous pages of results.
+
+::::{tab-set}
+
+:::{tab-item} Schema browser
+Click on schema elements to expand the tree, or use the '+' icon to expand all elements. Use { } to view the underlying schema for any section. Required fields are indicated in **bold**.
+
+ <script src="../_static/docson/widget.js" data-schema="../network-schema.json"></script> 
+:::
+
+:::{tab-item} Small file example
+The following example shows a network package containing two networks:
+```{jsoninclude} ../../examples/json/network-package.json
```

Comment:
I think it would be good to add a few example fields here, clarity

Reply:
Done in 2b4ae25476f61df5925ea849a54a3597d64ad6ab